props/prop_base.h

```diff
@@ -866,6 +878,23 @@ class PropBase : CommandParser, Looper, protected SaberBase {
   uint32_t last_on_time_;
 #endif
 
+#ifdef ENABLE_MENU_DIMBLADE
+  bool change_brightness_ = false;
+  bool BrightnessChangeMode(){return change_brightness_;}
```

Comment:
lots of spaces missing:
bool BrightnessChangeMode() { return change_brightness_; }